examples.json

```
@@ -0,0 +1,347 @@
+        "parameters": [
+          {
+            "name": "bookmarkId",
+            "in": "path",
+            "description": "ID of bookmark to return",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": { "$ref": "#/definitions/BookmarkRecord" }
+          },
+          "400": { "description": "Invalid ID supplied" },
+          "404": { "description": "Bookmark not found" }
+        },
+        "security": [{ "api_key": [] }]
+      },
+      "delete": {
+        "tags": ["bookmarks"],
+        "summary": "Deletes a bookmark",
+        "description": "",
+        "operationId": "deleteBookmark",
+        "produces": ["application/xml", "application/json"],
+        "parameters": [
+          {
+            "name": "api_key",
+            "in": "header",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "bookmarkId",
+            "in": "path",
+            "description": "Bookmark id to delete",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "400": { "description": "Invalid ID supplied" },
+          "404": { "description": "Bookmark not found" }
+        },
+        "security": [
+          { "bookmarks_auth": ["write:bookmarks", "read:bookmarks"] }
+        ]
+      }
+    }
+  },
+  "parameters": {
+    "BearerHeader": {
+      "in": "header",
+      "name": "Authorization",
+      "description": "JWT token required to get resources.",
+      "type": "string",
+      "required": true,
+      "default": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxIiwibmFtZSI6IkEiLCJpYXQiOjEsImVic2NvX3B1YV9pZCI6IjA3MjQyZjlhIn0.nRe7fIjMAIf1SkpFOTAEga3Gxx_Tk_QUgwATrgZpGfg"
+    },
+    "PersonalizedHeader": {
+      "in": "header",
+      "name": "X-EIS-ID-TOKEN",
+      "description": "Service identity header.",
+      "type": "string",
+      "required": true,
+      "default": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxIiwibmFtZSI6IkEiLCJpYXQiOjEsImVic2NvX3B1YV9pZCI6IjA3MjQyZjlhIn0.nRe7fIjMAIf1SkpFOTAEga3Gxx_Tk_QUgwATrgZpGfg"
+    }
+  },
+  "securityDefinitions": {
+    "bookmarks_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://bookmarksstore.swagger.io/oauth/authorize",
+      "flow": "implicit",
+      "scopes": {
+        "write:bookmarks": "modify bookmarks in your account",
+        "read:bookmarks": "read your bookmarks"
+      }
+    },
+    "api_key": { "type": "apiKey", "name": "api_key", "in": "header" }
+  },
+  "definitions": {
+    "BookmarkItem": {
+      "type": "object",
+      "properties": {
+        "an": {
+          "type": "string",
+          "example": "vmc.b30177686"
+        },
+        "contributors": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Contributors"
```

This should be singular `Contributor`, I think.
```suggestion
            "$ref": "#/definitions/Contributor"
```

---

done